dev/TabView/TabView.xaml

```diff
@@ -215,9 +215,9 @@
             <Setter.Value>
                 <ControlTemplate TargetType="local:TabViewItem">
 
-                    <Grid
-                        Background="{ThemeResource TabViewItemHeaderBackground}"
-                        Padding="{TemplateBinding Padding}">
+                    <Grid x:Name="LayoutRoot"
+                        Padding="{TemplateBinding Padding}"
+                        Margin="-1,0,0,0">
```

Comment:
>Margin="-1,0,0,0" [](start = 24, length = 17)

Is this so that the separator is only visible during drag operations?

Reply:
It's so that each item covers the previous item's separator. I can add a comment to explain.